x/dex/example/register-pair-proposal.json

```diff
@@ -6,18 +6,18 @@
         "contract_addr": "sei14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sh9m79m",
         "pairs": [
           {
-            "price_denom": "usdc",
-            "asset_denom": "sei",
+            "price_denom": "uusdc",
```

Comment:
kind of sad that json doesn't allow comments, but these denoms don't have to correspond to asset list denoms; they are denoms that the registering contract recognize internally